challenge_receptionist/src/challenge_receptionist/find_empty_seat2.py

```diff
@@ -0,0 +1,177 @@
+#! /usr/bin/env python
```

Comment:
This file is not needed unless you want to test this version and the original side by side. I think it was originally created as a tournament hack. But now you may feel free to make changes to find_empty_seat.py